src/ufuncs.c

```diff
@@ -3580,6 +3540,9 @@ int init_ufuncs(PyObject* m, PyObject* d) {
   DEFINE_YY_Y_REORDERABLE(union);
```

Comment:
@caspervdw this `_REORDERABLE` was to enable those to be used as reduction? In which case we can simpify this and make all set operations just `YY_Y`?

Reply:
Yes that's right.